proto/src/util.rs

```diff
@@ -76,7 +83,11 @@ impl TryFrom<Status> for grpcio::RpcStatus {
 
     fn try_from(value: Status) -> grpcio::Result<Self> {
         #[cfg(feature = "protobuf-codec")]
-        let details = protobuf::Message::write_to_bytes(&value)?;
+        let details = protobuf::Message::write_to_bytes(&value)
+            .map_err(|_err| grpcio::Error::RemoteStopped)?;
+        #[cfg(feature = "protobufv3-codec")]
+        let details = protobufv3::Message::write_to_bytes(&value)
```

Comment:
If protobufv3 is re-exported as protobuf, then here only needs to write `protobuf::Message::write_to_bytes`.

Reply:
Done